src/Core/Algorithms/Visualization/OsprayDataAlgorithm.cc

```diff
@@ -421,19 +567,28 @@ OsprayGeometryObjectHandle OsprayDataAlgorithm::fillDataBuffers(FieldHandle fiel
     }
   }
 
+    FieldInformation info(field);
+    
+    
   auto& index = fieldData.index;
   {
     for (const auto& face : facade->faces())
     {
       auto nodes = face.nodeIndices();
-      index.push_back(static_cast<int32_t>(nodes[0]));
-      index.push_back(static_cast<int32_t>(nodes[1]));
-      index.push_back(static_cast<int32_t>(nodes[2]));
+      if(info.is_quadsurfmesh()){
+        index.push_back(static_cast<int32_t>(nodes[3]));
```

Comment:
Add a comment explaining the inverse order of nodes added (3,2,1,0)

Reply:
I remember I had a discussion with Jess about that. When added in the given order the faces looked flipped (the normal doesn't look right for the light)